crates/muvm/src/guest/bin/muvm-guest.rs

```diff
@@ -13,6 +13,7 @@ use muvm::guest::sommelier::exec_sommelier;
 use muvm::guest::user::setup_user;
 use muvm::guest::x11::setup_x11_forwarding;
 use muvm::utils::env::find_in_path;
+use rustix::fd::AsFd;
```

Comment:
This is just a [re-export](https://docs.rs/rustix/latest/rustix/fd/trait.AsFd.html) of [`std::os::fd::AsFd`](https://doc.rust-lang.org/std/os/fd/trait.AsFd.html)